File: sample-code/fibonacci.php
<?php
// Fibonacci sequence generator (iterative)

function fib(int $n): array {
    $out = [];
    $a = 0;
    $b = 1;
    for ($i = 0; $i < max(0, $n); $i++) {
        $out[] = $a;
        $next = $a + $b;
        $a = $b;
        $b = $next;
    }
    return $out;
}

$n = 12;
echo implode(", ", fib($n)) . "\n";

File: sample-code/fibonacci.py
# Fibonacci sequence generator (iterative)

def fib(n: int) -> list[int]:
    out: list[int] = []
    a, b = 0, 1
    for _ in range(max(0, n)):
        out.append(a)
        a, b = b, a + b
    return out


if __name__ == "__main__":
    n = 12
    print(", ".join(map(str, fib(n))))
